app/forms/support_interface/application_forms/revert_withdrawal_form.rb

```diff
@@ -3,20 +3,33 @@ module ApplicationForms
     class RevertWithdrawalForm
       include ActiveModel::Model
 
-      attr_accessor :accept_guidance, :audit_comment_ticket
+      attr_accessor :accept_guidance, :audit_comment_ticket, :application_choice
 
-      validates :accept_guidance, :audit_comment_ticket, presence: true
+      validates :accept_guidance, presence: true
+      validate :valid_service
       validates_with ZendeskUrlValidator
 
-      def save(application_choice)
+      def save
         self.accept_guidance = ActiveModel::Type::Boolean.new.cast(accept_guidance)
 
-        return false unless valid?
+        validate
 
-        SupportInterface::RevertWithdrawal.new(
+        return false if errors.any?
+
+        revert_withdrawal_service.save
+      end
+
+      def revert_withdrawal_service
```

Comment:
Should this be a private method if you're only using it in this class?

Reply:
Makes sense, updated here.

https://github.com/DFE-Digital/apply-for-teacher-training/pull/9587/commits/4dc422eeb77d5c4c9f61f436761b942c76af5131